templates/file_crash_bug_description.md.jinja

```diff
@@ -0,0 +1,54 @@
+{#
+    Part of this template was extracted from Socorro's repository:
+    https://github.com/mozilla-services/socorro/blob/ff8f5d6b41689e34a6b800577d8ffe383e1e62eb/webapp/crashstats/crashstats/jinja2/crashstats/bug_comment.txt
+    https://github.com/mozilla-services/socorro/blob/ff8f5d6b41689e34a6b800577d8ffe383e1e62eb/webapp/crashstats/signature/jinja2/signature/signature_summary.html#L7-L41
+#}
+Crash report: https://crash-stats.mozilla.org/report/index/{{ uuid }}
+
+{% if java_stack_trace %}
+Java stack trace:
+```
+{{ java_stack_trace|truncate(5000, True)|safe }}
+```
+{% elif frames %}
+{% if moz_crash_reason %}
+MOZ_CRASH Reason: ```{{ moz_crash_reason|safe }}```
+{% elif reason %}
+Reason: ```{{ reason|safe }}```
+{% endif %}
+{% if crashing_thread is none %}
+No crashing thread identified; using thread 0.
+{% endif %}
+Top {{ frames|length }} frames of crashing thread:
+```
+{% for frame in frames -%}
+{{ frame.frame|safe}}  {{ frame.module|safe }}  {{ frame.signature|safe }}  {{ frame.source|safe }}
+{% endfor -%}
+```
+{% endif %}
+
+
+Here are some insights about the crash signature based on recent data at the time of reporting the bug:
+
+- **Crash kind:** {{ "Plugin Crash" if signature.is_plugin_crash  else  "Browser Crash" }}
+- **Is startup crash:** {{ "Yes - " if signature.is_startup_related_crash else "No" }}
+{%- if signature.is_startup_crash -%}
+all crashes happened during startup
+{%- elif signature.is_potential_startup_crash -%}
+{{ signature.num_startup_crashes }} out of {{ signature.num_crashes }} crashes happened during startup
+{%- elif signature.is_startup_window_crash  -%}
+more than half of the crashes happened during the first minute after launch
+{%- endif %}
+- **Has user comments:** {{ "Yes" if signature.has_user_comments else "No" }}
```

Comment:
What is the goal here? Let the developer know there are comments so they go look at them? Should we then have a link to the Socorro page to see the comments to simplify their life?

Reply:
Yes, that is correct. Done in b076aa469e91164c1e6aa78aca930a5a33bdd3cd.